### PR TITLE
pop-up

### PR DESCRIPTION
**Pull request by vorobeyka**
###pop-up

[Demo](https://vorobeyka.github.io/pop-up-init) |
[Code base](https://github.com/vorobeyka/pop-up-init/tree/master)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by babayK0** [comment]
@vorobeyka Hi :hand: 
Looks like your demo doesn't work, please fix this)

File: submissions/vorobeyka/pop-up/Index.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Popups</title>
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+<body>
+	<nav class="nav">
+		<ul class="menu">
```
Comment (babayK0) [inline submissions/vorobeyka/pop-up/Index.html]: Something goes wrong with your code formatting, please, use code formatter like a [Prettier](https://prettier.io/)

File: submissions/vorobeyka/pop-up/css/main.css [hunk 2]
```diff
@@ -0,0 +1,188 @@
+* {
+	font: small arial sans-serif;
```
Comment (babayK0) [inline submissions/vorobeyka/pop-up/css/main.css]: Use [Prettier](https://prettier.io/) for CSS too.

Comment (vorobeyka) [reply submissions/vorobeyka/pop-up/css/main.css]: Okay, thx for feedback.

**Comment by babayK0** [comment]
Looks like abandoned.

**Pull request closed without merging**